src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -1130,6 +1130,10 @@ def test_feature_list(self):
         # Once a feature goes GA , it will be removed from the feature list. Once that happens, use other ones to test
         self.cmd('feature show --namespace Microsoft.Network -n AllowLBPreview')
 
+    @AllowLargeResponse(8192)
+    def test_feature_unregister(self):
+        self.cmd('feature unregister --namespace Microsoft.Network --name AllowLBPreview', checks=self.check('properties.state', 'Unregistered'))
```

Comment:
Can we add some check for command response? Or it is possible to utilize `az feature show` to check state after unregistering?

Reply:
I might miss something. It does check the response with "checks=self.check('properties.state', 'Unregistered')". Anything else you think we should check here?

Reply:
yes. you can check the state in response.